now.json

```diff
@@ -16,7 +16,7 @@
     },
     {
       "src": "/graphql",
-      "dest": "https://todo/graphql"
+      "dest": "https://reddit-lite-graphql.now.sh"
```

Comment:
should be /graphql
<img width="1318" alt="Screen Shot 2019-05-15 at 6 26 57 PM" src="https://user-images.githubusercontent.com/4022631/57819879-293cb480-773f-11e9-9a94-ad6fc4e5670b.png">
<img width="1318" alt="Screen Shot 2019-05-15 at 6 25 32 PM" src="https://user-images.githubusercontent.com/4022631/57819882-2b9f0e80-773f-11e9-9d3a-fee697edf89e.png">

Reply:
https://reddit-lite-8xvmz7wo5.now.sh/?graphql=https://reddit-lite-graphql.now.sh/graphql